crates/oxc_linter/src/snapshots/check_property_names.snap

```diff
@@ -0,0 +1,111 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: check_property_names
+---
+  ⚠ eslint-plugin-jsdoc(check-property-names): No root defined for @property path.
+   ╭─[check_property_names.tsx:4:27]
+ 3 │                        * @typedef {SomeType} SomeTypedef
+ 4 │                        * @property Foo.Bar
+   ·                                    ───────
+ 5 │                        */
+   ╰────
+  help: @property path declaration `Foo.Bar` appears before any real property.
+
+  ⚠ eslint-plugin-jsdoc(check-property-names): No root defined for @property path.
+   ╭─[check_property_names.tsx:5:27]
+ 4 │                        * @property foo
+ 5 │                        * @property Foo.Bar
+   ·                                    ───────
+ 6 │                        */
+   ╰────
+  help: @property path declaration `Foo.Bar` appears before any real property.
+
+  ⚠ eslint-plugin-jsdoc(check-property-names): No root defined for @property path.
+   ╭─[check_property_names.tsx:5:36]
+ 4 │                        * @typedef {SomeType} SomeTypedef
+ 5 │                        * @property {string} employees[].name - The name of an employee.
+   ·                                             ────────────────
+ 6 │                        * @property {string} employees[].department - The employee's department.
+   ╰────
+  help: @property path declaration `employees[].name` appears before any real property.
+
+  ⚠ eslint-plugin-jsdoc(check-property-names): No root defined for @property path.
+   ╭─[check_property_names.tsx:6:36]
+ 5 │                        * @property {string} employees[].name - The name of an employee.
+ 6 │                        * @property {string} employees[].department - The employee's department.
+   ·                                             ──────────────────────
+ 7 │                        */
+   ╰────
+  help: @property path declaration `employees[].department` appears before any real property.
+
+  ⚠ eslint-plugin-jsdoc(check-property-names): Duplicate @property found.
+   ╭─[check_property_names.tsx:5:27]
+ 4 │                        * @property foo
+ 5 │                        * @property foo
```

Comment:
There's a trick for labelling multiple spans https://github.com/oxc-project/oxc/blob/6532799ff29bf39a562c0d09a83cea9c7f1ee142/crates/oxc_linter/src/rules/eslint/no_sparse_arrays.rs#L58-L63

Reply:
Wow, I didn't know that! 😮 => https://github.com/oxc-project/oxc/pull/2989/commits/d112048d599a8cf39b3fe244930456cc33e65ff3